readme.md

```diff
@@ -85,6 +85,7 @@ Full product comparison:
     <!-- -->
     <tr><td colspan="3"><h4>Additional Features</h4></td></tr>
     <tr><td><b>Incognito mode detection</b><br/><i>works in all modern browsers - see our full list of <a href="https://dev.fingerprintjs.com/docs/browser-support/" target="_blank">browsers supported</a></i></td><td align="center">–</td><td align="center">✓</td></tr>
+    <tr><td><b>Confidence score</b><br/><i>Indicates how accurate each identification event is</i></td><td align="center">–</td><td align="center">✓</td></tr>
```

Comment:
There is a confidence score in open agent. We can tell that the open score approximate and the pro score is accurate because it's based on the real statistics.